requirements.txt

```diff
@@ -1,9 +1,8 @@
 psutil~=5.8.0
 requests~=2.27.1
-grpcio~=1.43.0
+grpcio~=1.53.0
```

Comment:
The main motive for it is upgrading gprofiler to python 3.11 which requires newer grpcio.